frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -280,21 +281,28 @@ export const surveyLogic = kea<surveyLogicType>([
                                 WHERE event = 'survey shown'
                                     AND properties.$survey_id = ${props.id}
                                     AND timestamp >= ${startDate}
-                                    AND timestamp <= ${endDate}),
+                                    AND timestamp <= ${endDate}
+                                    AND {filters}),
```

Comment:
is there a world where `filters` is empty/null? if so, the SQL will break

Reply:
so the hogql will ignore empty filters and we don't have to do anything here?

Reply:
yes